PerfettoCds/Pipeline/CompositeDataCookers/PerfettoCpuSchedEventCooker.cs

```diff
@@ -29,32 +29,49 @@ public sealed class PerfettoCpuSchedEventCooker : CookedDataReflector, IComposit
         {
             PerfettoPluginConstants.ThreadCookerPath,
             PerfettoPluginConstants.ProcessCookerPath,
-            PerfettoPluginConstants.SchedSliceCookerPath
+            PerfettoPluginConstants.SchedSliceCookerPath,
+            PerfettoPluginConstants.FtraceEventCookerPath
         };
 
         [DataOutput]
         public ProcessedEventData<PerfettoCpuSchedEvent> CpuSchedEvents { get; }
 
+        [DataOutput]
+        public ProcessedEventData<PerfettoCpuWakeEvent> CpuWakeEvents { get; }
+
         public PerfettoCpuSchedEventCooker() : base(PerfettoPluginConstants.CpuSchedEventCookerPath)
         {
-            this.CpuSchedEvents =
-                new ProcessedEventData<PerfettoCpuSchedEvent>();
+            this.CpuSchedEvents = new ProcessedEventData<PerfettoCpuSchedEvent>();
+            this.CpuWakeEvents = new ProcessedEventData<PerfettoCpuWakeEvent>();
         }
 
         public void OnDataAvailable(IDataExtensionRetrieval requiredData)
         {
             // Gather the data from all the SQL tables
             var threadData = requiredData.QueryOutput<ProcessedEventData<PerfettoThreadEvent>>(new DataOutputPath(PerfettoPluginConstants.ThreadCookerPath, nameof(PerfettoThreadCooker.ThreadEvents)));
             var processData = requiredData.QueryOutput<ProcessedEventData<PerfettoProcessEvent>>(new DataOutputPath(PerfettoPluginConstants.ProcessCookerPath, nameof(PerfettoProcessCooker.ProcessEvents)));
+            //var instantData = requiredData.QueryOutput<ProcessedEventData<PerfettoInstantEvent>>(new DataOutputPath(PerfettoPluginConstants.InstantCookerPath, nameof(PerfettoInstantCooker.Instants)));
```

Comment:
remove this

Reply:
However, we should leave all the code in to expose Instants. If nobody asks for it (like in the current PR, we won't perform the query and process the data). However, it makes our API more complete and leaves the support in place in case anybody else needs it in the future or we need to switch to it for wake data.